packages/client/src/api/wargames_api/index.ts

```diff
@@ -332,7 +332,7 @@ export const initiateGame = (dbName: string): Promise<MessageInfoType> => {
     const messageInfoType: MessageInfoType = {
       ...wargame,
       _rev: undefined,
-      _id: dbDefaultSettings._id,
+      _id: new Date().toISOString(),
```

Comment:
Hmm, @lilitkarapetyan  - so we don't use an id of `settings` for all versions of the wargame after it has been initiated?